app/modules/ibridge/README.md

```diff
@@ -0,0 +1,12 @@
+iBridge Module
+==============
+
+Details information about the iBridge in the Mac.
+
+Database:
+* ibridge_boot_uuid - VARCHAR(255) - Boot UUID of iBridge
+* ibridge_build - VARCHAR(255) - iBridge OS build
+* ibridge_model_identifier - VARCHAR(255) - Model ID of iBridge
+* ibridge_model_name - VARCHAR(255) - Model name of iBridge
+* ibridge_serial_number - VARCHAR(255) - iBridge serial number
```

Comment:
Maybe just keep the ib_serial_number

Reply:
Removed prefixes, except for ibridge_serial_number